src/code/z_demo.c

```diff
@@ -1,99 +1,1649 @@
+
+            func_80146F5C(globalCtx);
+            break;
+
+        case 0x27:
+            gSaveContext.playerForm = PLAYER_FORM_ZORA;
+            break;
+
+        case 0x28:
+            csCtx->frames = cmd->startFrame - 1;
+            break;
+    }
+}
+#else
+static u16 D_801BB15C = 0xFFFF;
+void Cutscene_Command_Misc(GlobalContext* globalCtx2, CutsceneContext* csCtx, CsCmdBase* cmd);
+#pragma GLOBAL_ASM("asm/non_matchings/code/z_demo/Cutscene_Command_Misc.s")
+#endif
+
+// Command 0x97: Set Environment Lighting
+void Cutscene_Command_SetLighting(GlobalContext* globalCtx, CutsceneContext* csCtx, CsCmdEnvLighting* cmd) {
+    if (csCtx->frames == cmd->startFrame) {
+        if (cmd->setting != 0x20) {
+            globalCtx->envCtx.unk_C3 = cmd->setting - 1;
+            globalCtx->envCtx.unk_DC = 1.0f;
+        } else {
+            globalCtx->envCtx.unk_C3 = 0xFF;
+        }
+    }
+}
+
+// Command 0x12C: Play Background Music or Fanfare
+void Cutscene_Command_PlaySequence(GlobalContext* globalCtx, CutsceneContext* csCtx, CsCmdMusicChange* cmd) {
+    if (csCtx->frames == cmd->startFrame) {
+        func_801A2C88(cmd->sequence - 1);
+    }
+}
+
+// Command 0x12D: Stop Background Music or Fanfare
+void Cutscene_Command_StopSequence(GlobalContext* globalCtx, CutsceneContext* csCtx, CsCmdMusicChange* cmd) {
+    if ((csCtx->frames >= cmd->startFrame) && (cmd->endFrame >= csCtx->frames)) {
+        func_801A2D54(cmd->sequence - 1);
+    }
+}
+
+// Command 0x9C: Fade Background Music or Fanfare over duration
+void Cutscene_Command_FadeSequence(GlobalContext* globalCtx, CutsceneContext* csCtx, CsCmdMusicFade* cmd) {
+    if ((csCtx->frames == cmd->startFrame) && (csCtx->frames < cmd->endFrame)) {
+        u8 frameCount = cmd->endFrame - cmd->startFrame;
+
+        if (cmd->type == 2) {
+            Audio_QueueSeqCmd((frameCount << 0x10) | 0x110000FF);
+        } else {
+            Audio_QueueSeqCmd((frameCount << 0x10) | NA_BGM_STOP);
+        }
+    }
+}
+
+// Command 0x12E: Play Ambience music
+void Cutscene_Command_PlayAmbienceSequence(GlobalContext* globalCtx, CutsceneContext* csCtx, CsCmdBase* cmd) {
+    if (csCtx->frames == cmd->startFrame) {
+        // Audio_PlayNatureAmbienceSequence
+        // nightSeqIndex is be natureAmbienceId
+        func_801A4A28(globalCtx->soundCtx.nightSeqIndex);
+    }
+}
+
+// Command 0x130:
+void func_800EAD48(GlobalContext* globalCtx, CutsceneContext* csCtx, CsCmdBase* cmd) {
+    if (csCtx->frames == cmd->startFrame) {
+        // Audio_SetSfxReverbIndexExceptOcarinaBank
+        func_801A4428(2);
+    }
+}
+
+// Command 0x131:
+void func_800EAD7C(GlobalContext* globalCtx, CutsceneContext* csCtx, CsCmdBase* cmd) {
+    if (csCtx->frames == cmd->startFrame) {
+        // Audio_SetSfxReverbIndexExceptOcarinaBank
+        func_801A4428(1);
+    }
+}
+
+#ifdef NON_MATCHING
+// Matches, but I don't want to deal with bss reordering yet
```

Comment:
Personally I'd prefer to use the phrase `// needs in-function static bss`. Everywhere else that just needs bss use this phrase. Makes it easier to find them.